ui/cap-react/src/antd/schemas/Schemas.js

```diff
@@ -0,0 +1,263 @@
+                  >
+                    <InfoCircleOutlined />
+                  </Tooltip>
+                </Col>
+                <Col xs={15} md={6}>
+                  {schema && (
+                    <Select
+                      style={{
+                        width: "100%",
+                      }}
+                      options={options}
+                      onChange={handleSelectChange}
+                      defaultValue="all"
+                    />
+                  )}
+                </Col>
+                <Col>
+                  <Tooltip
+                    placement="bottom"
+                    title={
+                      jsonError
+                        ? "Please fix the syntax errors to be able to view the diff"
+                        : selection === FULL_SCHEMA
+                          ? "View full diff"
+                          : "View field diff"
+                    }
+                  >
+                    <Button
+                      icon={<DiffOutlined />}
+                      onClick={() => setShowModal(true)}
+                      disabled={jsonError}
+                    />
+                  </Tooltip>
+                </Col>
+                <Col>
+                  <Tooltip
+                    placement="bottom"
+                    title={
+                      selection === FULL_SCHEMA
+                        ? "Revert all unsaved changes"
+                        : "Revert unsaved changes to this field"
+                    }
+                  >
+                    <Button icon={<UndoOutlined />} onClick={handleRevert} />
+                  </Tooltip>
+                </Col>
+                <Col>
+                  <Tooltip
+                    placement="bottom"
+                    title={
+                      selection === FULL_SCHEMA
+                        ? "Save all changes"
+                        : !EDITABLE_FIELDS.includes(selection)
+                          ? "Please select an editable field to be able to edit and save the changes"
+                          : "Save changes to this field"
+                    }
+                  >
+                    <Button
+                      icon={<SaveOutlined />}
+                      type="primary"
+                      onClick={handleSave}
+                      disabled={
+                        !EDITABLE_FIELDS.includes(selection) &&
+                        selection != FULL_SCHEMA
+                      }
+                    />
+                  </Tooltip>
+                </Col>
+              </Row>
+              <Row justify="center">
+                <Col xs={24}>
+                  <JSONInput
+                    width="100%"
+                    height="calc(100vh - 19em)"
+                    placeholder={field}
+                    onChange={handleEdit}
+                    viewOnly={!EDITABLE_FIELDS.includes(selection)}
+                    theme="light_mitsuketa_tribute"
```

Comment:
we can have the string as a const and customize if required. something like this.

```suggestion
                   const THEME = "light_mitsuketa_tribute"
                   theme={THEME}
```

Reply:
Not needed anymore since I changed the library we use to display the json :)